src/structures/MessageMentions.js

```diff
@@ -125,6 +125,12 @@ class MessageMentions {
     } else {
       this.crosspostedChannels = new Collection();
     }
+
+    /**
+     * The author of the message that this message is a reply to
```

Comment:
This sounds a bit odd imo
```suggestion
     * The author of the message that this message was replied to
```